docs/source/installed_packages.rst

```diff
@@ -30,6 +30,9 @@ imports, namespace packages, and custom import hooks.
 If you do not want to use typed packages, use the ``--no-site-packages`` flag
 to disable searching.
 
+Note that stub-only packages cannot be used with ``MYPYPATH``. If you want mypy
+to find the package, it must be installed.
```

Comment:
Can you clarify either here or somewhere else on this page what it means for a package to be "stub-only"? E.g. is a stub-only package a package that contains only `.pyi` files, or is a stub-only package one that has a name ending in `-stub`, contains only `.pyi` files, and contains a file named `py.typed`? 

(And if you do decide to add the exact definition somewhere else on the page, we should probably link to that here.)

Similarly, maybe clarify what it means for a package to be "installed"?

Reply:
I think ideally the docs should have a glossary, and this would be good to add there. I think linking to PEP 561 https://www.python.org/dev/peps/pep-0561/#stub-only-packages in this case is enough.